Src/PRuntimes/PSymbolicRuntime/src/main/java/psymbolic/runtime/Concretizer.java

```diff
@@ -250,12 +252,13 @@ public static GuardedValue<? extends PValue<?>> concretizePType (Object valueSum
 
     /**
      * Get a list of concrete values for the arguments
+     * @param print Enable printing of concrete value
      * @param pc Guard under which to concretize values
      * @param stop specifies when to stop getting more concrete values
      * @param args arguments
      * @return list of concrete values for arguments
      */ 
-    public static List<GuardedValue<List<Object>>> getConcreteValues(Guard pc, Predicate<Integer> stop, Function<ValueSummary, GuardedValue<?>> concretizer, ValueSummary ... args) {
+    public static List<GuardedValue<List<Object>>> getConcreteValues(boolean print, Guard pc, Predicate<Integer> stop, Function<ValueSummary, GuardedValue<?>> concretizer, ValueSummary ... args) {
```

Comment:
Would it be possible to move this as the last parameter in the function and also control this based on the verbosity input command?

Reply:
I removed the parameter completely, and instead added a static flag in [Concretizer](https://github.com/aman-goel/P/blob/cd486d40c187e6a897d3ec14e0b8e0c42da452c2/Src/PRuntimes/PSymbolicRuntime/src/main/java/psymbolic/runtime/Concretizer.java#L17) that is controlled through [verbosity flag](https://github.com/aman-goel/P/blob/cd486d40c187e6a897d3ec14e0b8e0c42da452c2/Src/PRuntimes/PSymbolicRuntime/src/main/java/psymbolic/commandline/EntryPoint.java#L81) to trigger verbose printing in all concrete enumeration scenarios.